### PR TITLE
Update App.svelte: Design -> Schema

### PR DESCRIPTION
**Pull request by robinlovelace-ate**
As per comments here https://github.com/acteng/atip/pull/145#issuecomment-1533573887 I think schema is a more useful word than Design here, even though it does affect the UI design.

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -89,9 +89,9 @@
     <label>
       ATIP schema:
```
Comment (dabreegster) [inline src/App.svelte:90]: We already label all of these things as schemas. So "ATIP Schema: Schema (housing)" feels redundant -- why not just "ATIP Schema: housing"?

Comment (robinlovelace-ate) [reply src/App.svelte:90]: > We already label all of these things as schemas. So "ATIP Schema: Schema (housing)" feels redundant -- why not just "ATIP Schema: housing"?

Oh yeah, good point.

Comment (robinlovelace-ate) [reply src/App.svelte:90]: Updated.

**Review by dabreegster — APPROVED** [review]
LGTM, thanks!

**Pull request merged**